website/docs/reference/events.mdx

```diff
@@ -0,0 +1,1071 @@
+
+## Event Log
+
+The Event Log lists all events in your Unleash instance. You can search and filter events by date range, event type, project, feature flag, and user. You can also export events as CSV or JSON files.
+
+To access events in the Admin UI, you need Admin access. To view all events, go to Admin > Event Log. To view events for a specific feature flag, go to the Event Log tab on that feature flag's page.
+
+Each [event type](#event-types) contains different data, but all events follow the same schema:
+
+
+| Field              | Type   | Description                                                       |
+|--------------------|--------|-------------------------------------------------------------------|
+| `id`*              | number | Unique identifier for the event; incremental whole number.        |
+| `type`*            | string | Describes the type of event (e.g., `feature-strategy-update`).    |
+| `createdAt`*       | string | Timestamp when the event was created; ISO 8601 format.            |
+| `createdBy`*       | string | Email of the user who triggered the event.                        |
+| `createdByUserId`* | number | Unique identifier of the user who triggered the event.            |
+| `data`             | object | Details about the event (e.g., feature strategy, API token data). |
+| `preData`          | object | Previous state before the event.                                  |
+| `tags`             | array  | Array of tags associated with the event.                          |
+| `featureName`      | string | Name of the feature flag associated with the event.               |
+| `project`          | string | Name of the project involved in the event.                        |
+| `environment`      | string | Environment related to the event (e.g., `development`).           |
+| `label`            | string | Brief description of the event.                                   |
+| `summary`          | string | Markdown formatted summary of the event.                          |
+
+
+Fields marked with an asterisk (*) are always returned in the event schema. Other fields may be optional or have null values, depending on the event type.
+
+See the [feature-created event type](#feature-created) for an example event.
+
+
+## Event Timeline
+
+:::note Availability
+
+**Plan**: [Enterprise](https://www.getunleash.io/plans/enterprise) | **Version**: `6.3+` in BETA
+
+:::
+
+Event Timeline highlights recent events across all your projects in one unified timeline. You can access the event timeline from the top menu in the Admin UI.
+
+![Event timeline in Unleash Admin UI](/img/event-timeline.png)
+
+The timeline shows key events and signals for up to 48 hours per environment and is designed to help you get an overview of changes and quickly identify and debug any issues.
```

Comment:
I think we should highlight that the timeline displays signals and link to the signals documentation for more context.

Depending on what you find relevant, we could also mention:
- The timeline is toggleable at any time
- Hovering over an event shows detailed information
- Events that occur close together are grouped
- The timeline remembers your previous selections
- You can switch between time spans (e.g., last 30 min, last hour)
- You can filter by environment (we default to the first active production environment)
- Environment-agnostic events are always displayed (e.g., archiving a feature flag)
- We only display key event types

Reply:
7bba5f0862c88439733dd3c63197934bb6a2b412